kzg_prover/src/utils/dummy_entries.rs

```diff
@@ -0,0 +1,102 @@
+use num_bigint::BigUint;
+use rand::{distributions::Alphanumeric, Rng};
+use rayon::prelude::*;
+use std::error::Error;
+
+use crate::cryptocurrency::Cryptocurrency;
+use crate::entry::Entry;
+
+// This is for testing purposes with a large dataset instead of using a CSV file
+pub fn generate_dummy_entries<const N_CURRENCIES: usize>(
+    entries: &mut [Entry<N_CURRENCIES>],
```

Comment:
In this case it should be ok to return the (entries, currencies) from the function instead of passing in and filling the mutable arrays (which is used in parser to save the memory).